crowbar_framework/config/repos.yml

```diff
@@ -0,0 +1,173 @@
+---
+suse-12.0:
+  repos:
+    sles12-pool:
+      name: "SLES12-Pool"
+      required: "mandatory"
+      product_name: "common"
+      content_key:
+      repomd:
+        key:
```

Comment:
So this is the name of the file containing the key (which is not what I was expecting, but that's okay). I don't think we need an attribute for this as this will always be the same value anyway.